types/select/index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare function select(element: HTMLElement): void;
+
+export = select;
```

Comment:
this one is shipped with UMD module IMO:
https://unpkg.com/browse/select@1.1.2/dist/select.js
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
so you would probably have to apply UMD module template exports